### ISSUE TITLE
Covers validator not working

### ISSUE THREAD

**Issue by theofidry** [oradwell/covers-validator#28]
When upgrading to 1.2.0 with PHPUnit 9.3.7 I can no longer get covers-validator working:

```
PHP Fatal error:  Uncaught Error: Class 'PHPUnit\TextUI\Configuration\Loader' not found in /Users/tfidry/Project/Alice/nelmio/vendor-bin/covers-validator/vendor/ockcyp/covers-validator/src/Loader/ConfigLoader.php:30
```

See https://travis-ci.org/github/nelmio/alice/jobs/720270946

**Comment by oradwell** [comment]
these files are now phpunit internals. they may have changed. I'd suggest phar build of covers-validator to see if the issue persists

**Comment by gbirke** [comment]
The class has moved from `PHPUnit\TextUI\Configuration\Loader` to `PHPUnit\TextUI\XmlConfiguration\Loader` in PHPUnit 9.3 (changed from PHPUnit 9.2), see https://github.com/sebastianbergmann/phpunit/commit/516b3080b85641219fe322b6d72977829deccd24#diff-ed5b387b42f9b89f8157fccf54b6132b

We are in the situation that this tool offers a wide range of PHPUnit compatibility and uses classes that are internal to PHPUnit - for a good reason, because the PHPUnit Config loader is quite complex. I can see several ways going forward:

* As a quick fix, change composer.json to indicate that `covers-validator` is only compatible with PHPUnit up to 9.2.x
* Synchronize `covers-validator` releases with the release cycle of PHPUnit
* Do some more `class_exists` trickery in `Loader/ConfigLoader.php`. I could provide a patch for that, if you decide that's the best way forward for this project, @oradwell

**Comment by theofidry** [comment]
I think if it's just the class moving implementing a patch shouldn't be too much of a problem.

But overall I believe it's a bit of a shame that PHPUnit does not do that* and instead we need to rely on a tool itself relying on quite some internal elements of PHPUnit.

*: it actually does it, but at the end of the run which makes it non-ideal when looking for fast feedback

**Comment by oradwell** [comment]
@gbirke @theofidry yeah, `class_exists` trickery should be quite simple. it looks much simpler than the 9.x change. I'd really appreciate it if you could create a patch for it @gbirke

**Comment by oradwell** [comment]
Released in v1.3.0 https://github.com/oradwell/covers-validator/releases/tag/v1.3.0
